### PR TITLE
Add ModuleInfo for torch.nn.ChannelShuffle

### PR DESCRIPTION
**Pull request by isdanni**
Summary:
Add ModuleInfo for torch.nn.ChannelShuffle:

https://github.com/pytorch/pytorch/pull/105351#pullrequestreview-1533879586

Test Plan: Please see GitHub Actions.

Differential Revision: D48021100

### CODE REVIEW

File: torch/testing/_internal/common_modules.py [hunk 1]
```diff
@@ -771,6 +771,22 @@ def module_inputs_torch_nn_BatchNorm3d(module_info, device, dtype, requires_grad
                     forward_input=FunctionInput(make_input((0, 5, 2, 2, 2))),
                     desc='zero_batch')]
 
+def module_inputs_torch_nn_ChannelShuffle(module_info, device, dtype, requires_grad, training, **kwargs):
+    make_input = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    return [
+        ModuleInput(constructor_input=FunctionInput(2),
```
Comment (mikaylagawarecki) [inline torch/testing/_internal/common_modules.py:778]: This sample input where you pass in a singleton tensor is causing the failure, seems like there is a `TORCH_CHECK` that ensures input size is at least 2 [here](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/ChanelShuffle.cpp#L36-L38)

**Review by mikaylagawarecki — COMMENTED** [review]
Thanks @isdanni, added a comment on how to fix CI :)

**Comment by isdanni** [comment]
So sorry for the delay! I'm working on fixing this. The Internal Sync check might be a bit hard to bypass. If it's not possible to fix I will open a new one.

**Pull request closed without merging**